support/schemas/user.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This isn't valid JSON-API - take a look at the JSON API Spec and some of the schemas for the other resources to get a good idea of what this should look like.